main.rb

```diff
@@ -0,0 +1,576 @@
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
+end
+def bar
+  a = "aaa"
+  return a
+end
+def baz
+  a = "aaa"
+  return a
+end
+def foo
+  a = "aaa"
+  return a
```

Comment:
__[development][rubocop]__
* Method `Object#bar` is defined at both /sideci/sandbox2/main.rb:6 and /sideci/sandbox2/main.rb:51.
* Redundant `return` detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-explicit-return)